src/main/java/com/sap/oss/phosphor/fosstars/nvd/data/CpeMatch.java

```diff
@@ -1,98 +1,67 @@
 package com.sap.oss.phosphor.fosstars.nvd.data;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
-import java.util.ArrayList;
-import java.util.List;
 import java.util.Optional;
+import javax.annotation.Nullable;
 import org.apache.commons.lang3.StringUtils;
 
 /**
  * CPE match string or range.
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @JsonPropertyOrder({
-    "vulnerable",
     "cpe22Uri",
     "cpe23Uri",
-    "versionStartExcluding",
-    "versionStartIncluding",
     "versionEndExcluding",
```

Comment:
I think this should be "versionStartIncluding"

Reply:
Yes, `JsonPropertyOrder` should contains "including" ones.